data/avro-schema/src/main/resources/avro/net/corda/data/membership/rpc/request/AddApprovalRuleRequest.avsc

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "record",
+  "name": "AddApprovalRuleRequest",
+  "namespace": "net.corda.data.membership.rpc.request",
```

Comment:
Commented on the runtime-os PR, but I don't think we need the RPC avro schemas here. (See runtime os comment for why)

Reply:
Removed in [0a97f59](https://github.com/corda/corda-api/pull/790/commits/0a97f59720278f7640e0e85d36a2794a74937722)